build.gradle.kts

```diff
@@ -16,8 +16,11 @@ repositories {
 }
 
 dependencies {
-	implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
 	implementation("org.springframework.boot:spring-boot-starter-batch")
+	implementation("org.springframework.retry:spring-retry:1.2.4.RELEASE")
```

Comment:
Can we remove this version and let the dependency resolver sort the version for us?

Reply:
Yup that's good.